docs/data/base/components/slider/UnstyledSliderBasic/tailwind/index.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import { useTheme } from '@mui/system';
+import Slider, { SliderThumbSlotProps } from '@mui/base/Slider';
+
+function useIsDarkMode() {
+  const theme = useTheme();
+  return theme.palette.mode === 'dark';
+}
+
+const Thumb = React.forwardRef<HTMLSpanElement, SliderThumbSlotProps>(function Thumb(
+  props,
+  ref,
+) {
+  const { ownerState, className, ...other } = props;
+  const focusVisible = props['data-index'] === ownerState.focusedThumbIndex;
+  const active = props['data-index'] === ownerState.activeThumbIndex;
+  return (
+    <span
+      ref={ref}
+      className={clsx(
+        `absolute w-4 h-4 -ml-1.5 -mt-1.5 box-border rounded-full outline-0 border-3 border-solid border-current bg-white hover:shadow-outline-purple ${
+          focusVisible || active ? 'shadow-outline-purple' : ''
```

Comment:
I didn't want to use utils from our system, like the alpha, so I used the same shadow outline used in the other component. @danilo-leal @zanivan let me know if you think we should maybe add additional one, for distinguishing the styles between focus visible, active and hover.

Reply:
I don't think it's necessary to have more, as we don't have it, neither on plain CSS nor system.

Regarding the shadow-outline, on the switch, we're using rgba instead of hex value, so I converted the purple hex to rgb, and added the same 0.25 for the alpha. Would it be a problem? I thought it'd look more consistent with the other styles here.